directory/onion_directory.py

```diff
@@ -110,14 +109,17 @@ def handle_path_request(conn):
     data = conn.recv(1024).decode('utf-8').rstrip()
 
     # TODO: read path request, decrypt, enrypt with symkey
-    conn.sendall( str.encode(str(get_path())) )
-    conn.sendall( str.encode(" " + data.upper()) )
+    conn.sendall( json.dumps(get_path()).encode('utf-8') )
 
     conn.close()
 
 
 def get_path():
-    return random.sample(NODES, 3)
+    #return random.sample(NODES, 3)
+    # TODO implement actual retrieval of data
+    return [{'addr':'lab2-1','key':'KEY_1'},
+            {'addr':'lab2-2','key':'KEY_2'},
+            {'addr':'lab2-3','key':'KEY_3'}]
 
```

Comment:
I don't see why this change was necessary

Reply:
it is not a change, rather a stub based on the specifications for the directory node:
it should return 3 addresses and their keys.
Coding that functionality isn't part of the scope of this PR-issue

Reply:
it seems you changed the stub from doing `random.sample(NODES, 3)` to 
```python
return [{'addr':'lab2-1','key':'KEY_1'},
        {'addr':'lab2-2','key':'KEY_2'},
        {'addr':'lab2-3','key':'KEY_3'}]
```

but ya I guess it's just a formatting thing